source/optional_features/welsh_language/index.html.md.erb

```diff
@@ -19,10 +22,7 @@ tool to add a Welsh service name to your service.
 If you add a Welsh service name to your service, users will see this on Welsh-language
 payment pages instead of the English service name.
 
-When users make a payment, they will also [see a
-`description`](/payment_flow/#making-a-payment). If you use Welsh-language
-payment pages, you may also want to use Welsh text in the
-`description`.
+If you're integrating with the GOV.UK Pay API, you may also want to use Welsh text in the [`description`](/payment_flow/#making-a-payment). Users will see the Welsh `description` when they make a payment.
```

Comment:
This sort of applies to payment links too — the title the service sets for the payment link becomes the description and they should set this as Welsh. Not sure if it’s worth calling this out because the payment links setup flow makes it pretty clear you should be entering Welsh text.

Reply:
> I wonder if it might be worth having separate sections for Welsh payments created by an API integration and Welsh payment links to make it clear that different considerations apply to each (e.g. the `description` advice only directly applies to API integrations while setting a Welsh service name also affects the URL with payment links).

@katiebates @alexbishop1 Thanks for the feedback!

I followed up with Tracy, and we realised I was complicating things by trying to use the Welsh page to detail how to set up payments links. That's not really a job for the tech docs. So in https://github.com/alphagov/pay-tech-docs/pull/288/commits/b6ef9b803bb04160057750ce3ffd1f822a8c9e6f I've pared things back - so we simply mention payment links are available in Welsh, then guide users to https://www.payments.service.gov.uk/payment-links/ as normal.

Tracy will highlight the need to set up a Welsh service name *before* you create a Welsh payment link, in content inside the admin tool itself or on the payment links product page.